### ISSUE TITLE
session statistics

### ISSUE THREAD

**Issue by daviddalpiaz** [PrairieLearn/PrairieTest-feedback#2]
it would be helpful to have some simple statistics during exam sessions like "number of students currently checked-in" for comparison to number of zoom participants.

**Comment by Jim-Sosnowski** [comment]
it would helpful if course staff could specifically see which individual students in their courses checked-in

**Comment by joelostblom** [comment]
I agree that showing a count for the number of students checked in and not checked in (or even just numbering the students in the table would be helpful when doing a headcount of the student in the classroom to make sure it matches with who is checked in.
